core/services/assets/Registry.php

```diff
@@ -634,8 +636,10 @@ private function removeAlreadyRegisteredDataForScriptHandle($script_handle)
         if (isset($this->script_handles_with_data[$script_handle])) {
             global $wp_scripts;
             if ($wp_scripts->get_data($script_handle, 'data')) {
-                unset($wp_scripts->registered[$script_handle]->extra['data']);
-                unset($this->script_handles_with_data[$script_handle]);
+                unset(
+                    $wp_scripts->registered[$script_handle]->extra['data'],
+                    $this->script_handles_with_data[$script_handle]
+                );
```

Comment:
k this is a bit alarming because it seems to indicate that the branch this is in is out of date. The actual code in this method _currently_ within master and `FET/11400/block-manager` is:

```
private function removeAlreadyRegisteredDataForScriptHandle($script_handle)
    {
        if (isset($this->script_handles_with_data[$script_handle])) {
            global $wp_scripts;
            $unset_handle = false;
            if ($wp_scripts->get_data($script_handle, 'data')) {
                unset($wp_scripts->registered[$script_handle]->extra['data']);
                $unset_handle = true;
            }
            //deal with inline_scripts
            if ($wp_scripts->get_data($script_handle, 'before')) {
                unset($wp_scripts->registered[$script_handle]->extra['before']);
                $unset_handle = true;
            }
            if ($wp_scripts->get_data($script_handle, 'after')) {
                unset($wp_scripts->registered[$script_handle]->extra['after']);
            }
            if ($unset_handle) {
                unset($this->script_handles_with_data[$script_handle]);
            }
        }
    }
```

So we've got a problem if its not this in the branch you are editing.  There will be some nasty conflicts to try to resolve because of the additional changes you've made in this class in this branch (and possible reverts of work could happen too)

Reply:
branch just needed updating. is correct now